randovania/generator/filler/player_state.py

```diff
@@ -219,6 +219,26 @@ def filter_usable_locations(self, locations_weighted: WeightedLocations) -> Weig
 
         return locations_weighted
 
+    def should_have_hint(self, pickup: PickupEntry, current_uncollected: UncollectedState,
+                         all_locations_weighted: WeightedLocations) -> bool:
+
+        if not pickup.item_category.is_major:
```

Comment:
shouldn't this be checking the interesting pickup thing?

Reply:
If it was picked at all it's an interesting pickup.